### PR TITLE
Added capability to send status of a worker.

### PR DESCRIPTION
**Pull request by yojagad**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    public WorkerStatusRequestHandler() {
+        super(StreamingMessage::getWorkerStatusRequest,
+              WorkerStatusResponse::newBuilder,
+              null,
+              StreamingMessage.Builder::setWorkerStatusResponse);
+    }
+
+    @Override
+    String execute(WorkerStatusRequest request, WorkerStatusResponse.Builder response) {
+        return "";
```
Comment (mathewc) [inline src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java:15]: Can we write a completed message like the other handlers do, e.g. `return String.format("WorkerStatusRequest completed");`?

**Comment by amamounelsayed** [comment]
@yojagad can you please check the tests?

File: src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    public WorkerStatusRequestHandler() {
+        super(StreamingMessage::getWorkerStatusRequest,
+              WorkerStatusResponse::newBuilder,
+              null,
+              StreamingMessage.Builder::setWorkerStatusResponse);
+    }
+
+    @Override
+    String execute(WorkerStatusRequest request, WorkerStatusResponse.Builder response) {
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java:14]: Please correct me if I am wrong, this message will not matter as the WorkerStatusResponse is empty in the protobuf message.

Comment (yojagad) [reply src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java:14]: True, but just leaving it there for future too when we actually update it with contents.

**Comment by yojagad** [comment]
@amamounelsayed Just squashed the commits. Please merge it whenever you can , thank you!

**Pull request merged**